troposphere/static/js/components/modals/migrate_resources/ResourceSelectMenu.jsx

```diff
@@ -22,7 +22,7 @@ const ResourceSelectMenu = React.createClass({
     },
     renderSelectMenu() {
         let { projects } = this.props;
-        let { resource, project } = this.props;
+        let { project } = this.props;
```

Comment:
It looks like this `let` could be included in the destructure of `this.props` as well:
```
let optionName = this.props.optionName;
```